tests/testCheckboxMultiField.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+class CheckboxMultiFieldTestCase extends WP_UnitTestCase {
+
+	function setUp() {
+		parent::setUp();
+
+		$args = array(
+			'post_author'  => 1,
+			'post_status'  => 'publish',
+			'post_content' => rand_str(),
```

Comment:
🔢 Random strings can create issues so you're better off changing it to a fixed string instead.